Makefile.dep

```diff
@@ -31,6 +31,17 @@ ifneq (,$(filter gnrc_uhcpc,$(USEMODULE)))
     USEMODULE += fmt
 endif
 
+ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
+    USEMODULE += softdevice_handler
+    USEMODULE += ble_common
+    USEMODULE += ble_6lowpan
```

Comment:
ble_6lo/ble_sixlo might be less confusing, because as far as I know isn't IPv6 over BLE called 6LoWPAN. ;-)